### PR TITLE
tweak to file adapter to fetch files from url

### PR DESCRIPTION
**Pull request by kirillseva**

### CODE REVIEW

File: R/adapter.r [hunk 1]
```diff
@@ -151,9 +151,11 @@ construct_file_adapter <- function() {
     if ('.rds' == substring(opts$resource, nchar(opts$resource) - 3, nchar(opts$resource)))
       readRDS(opts$resource)
     else {
+      test_if_url <- function(str) grepl("^https?://", str)
+      filesource <- ifelse(test_if_url(opts$resource), getURL(opts$resource), opts$resource)
```
Comment (robertzk) [inline R/adapter.r]: Doesn't `getURL` retrieve the actual content? That won't work as a `file` key below. Also, `read.csv` uses a fileConnection, which does URL parsing by default. For example, `read.csv("http://therobert.org/stuff/wizzo/iris.csv")` works for me.

Comment (kirillseva) [inline R/adapter.r]: https://github.com/kirillseva/titanic/blob/master/titanic.R#L6-L9

```url %>% getUrl %>% read.csv(text=.)``` is the combination that works for me

Comment (robertzk) [inline R/adapter.r]: Ahh, I see. It gets passed to `file` below when it should be passed to `text`. I thought I had tested the file adapter and it worked with URLs. :/

Doesn't matter! I will re-factor this and make syberiaStages obsolete today.

Comment (kirillseva) [inline R/adapter.r]: :+1:

**Pull request merged**